server/plugins/__tests__/airbrake-schema.spec.js

```diff
@@ -0,0 +1,18 @@
+const Joi = require('joi')
+const { options } = require('../airbrake-schema')
+
+describe('airbrake-schema', () => {
```

Comment:
I'd only done that file as it seemed simple to do, I think Matthew mentioned the normal airbrake file being awkward to test so just assumed that meant it could be left (for now)